client/layout/activity-panel/panels/inbox.js

```diff
@@ -3,44 +3,120 @@
  * External dependencies
  */
 import { __ } from '@wordpress/i18n';
+import { Button } from '@wordpress/components';
 import { Component, Fragment } from '@wordpress/element';
 import Gridicon from 'gridicons';
 
 /**
  * Internal dependencies
  */
-import ActivityCard from '../activity-card';
+import { ActivityCard, ActivityCardPlaceholder } from '../activity-card';
 import ActivityHeader from '../activity-header';
+import { getAdminLink } from 'lib/nav-utils';
 import { Section } from 'layout/section';
 
+const demoNotices = [
+	{
+		id: 1,
+		type: 'informational',
+		title: __( 'Accept Apple Pay using Stripe', 'wc-admin' ),
+		content: __(
+			'Your Stripe payment gateway now allows your customers to pay using Apple Pay.',
+			'wc-admin'
+		),
+		icon: 'customize',
+		status: 'unread',
+		source: 'woocommerce-core',
+		date_created: '2018-07-12T16:23:08Z',
+	},
+	{
+		id: 2,
+		type: 'warning',
+		title: __( 'Extension subscription expired', 'wc-admin' ),
+		content: __(
+			'Your subscription for WooCommerce Subscriptions expired on Jun 7th 2018.',
+			'wc-admin'
+		),
+		icon: 'notice-outline',
+		status: 'read',
+		source: 'woocommerce-core',
+		date_created: '2018-07-11T02:49:00Z',
+	},
+	{
+		id: 3,
+		type: 'informational',
+		title: __( 'Looking for the Store Notice setting?', 'wc-admin' ),
+		content: __( 'It can now be found in the customizer.', 'wc-admin' ),
+		icon: 'info-outline',
+		status: 'read',
+		source: 'woocommerce-core',
+		date_created: '2018-07-10T02:49:00Z',
+		actions: [
+			{
+				id: 1,
+				name: 'store-customizer-setting',
+				label: 'Open Customizer',
+				// @todo What format will links be? Should they default to external/full URLs?
+				// Full URLs that are wc-admin links will force a page refresh though…
+				url: getAdminLink( 'customize.php?autofocus%5Bpanel%5D=woocommerce' ),
+			},
+		],
+	},
+];
+
 class InboxPanel extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			loading: true,
+			notices: [],
+		};
+
+		setTimeout( () => {
```

Comment:
I know this is temporary, but just wanted to note that this can cause an issue if you switch tabs while it's still loading (so the panel gets unmounted).

`Warning: Can't call setState (or forceUpdate) on an unmounted component. This is a no-op, but it indicates a memory leak in your application. To fix, cancel all subscriptions and asynchronous tasks in the componentWillUnmount method.
    in InboxPanel (created by ActivityPanel)`

Reply:
Yeah I think it is fine to leave it since it is temporary code. But also fine if you want to fix it Kelly

Reply:
I'm not sure how long until we get anything for the notifications API, since there isn't a swagger project for it yet, so I don't know how long this placeholder will be here. I'll fix it here before merge